### PR TITLE
MAGECLOUD-3871: Write cloud.log in debug level to file

### PR DESCRIPTION
**Pull request by oshmyheliuk**
## Purpose of this pull request

This pull request will update information about MIN_LOGGING_LEVEL variable according to the latest code changes.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/cloud/env/variables-global.html
- https://devdocs.magento.com/guides/v2.3/cloud/env/variables-global.html

## Related PR
https://github.com/magento/ece-tools/pull/589

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/cloud/env/variables-global.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ stage:
     MIN_LOGGING_LEVEL: debug
 ```
 
+{: .bs-callout .bs-callout-warning }
+Doesn't effect log level for the file handler which is debug by default.
```
Comment (meker12) [inline guides/v2.2/cloud/env/variables-global.md]: The `MIN_LOGGING_LEVEL` setting does not change the log level configuration for the file handler, which is set to `debug` by default.

Comment (meker12) [inline guides/v2.2/cloud/env/variables-global.md]:
```suggestion
The setting for the`MIN_LOGGING_LEVEL` variable does not change the log level configuration for the file handler, which is set to debug by default.
```

Comment (meker12) [inline guides/v2.2/cloud/env/variables-global.md]:
```suggestion
The setting for the`MIN_LOGGING_LEVEL` variable does not change the log level configuration for the file handler, which is set to debug by default.
```

Comment (oshmyheliuk) [reply guides/v2.2/cloud/env/variables-global.md]: Thank you!

File: guides/v2.2/cloud/env/variables-global.md [hunk 2]
```diff
@@ -29,6 +29,9 @@ stage:
     MIN_LOGGING_LEVEL: debug
 ```
 
+{: .bs-callout .bs-callout-warning }
```
Comment (meker12) [inline guides/v2.2/cloud/env/variables-global.md]:
```suggestion
{: .bs-callout-warning }
```

**Pull request merged**